src/components/Join.js

```diff
@@ -103,59 +104,38 @@ const Consent = compose(
               contrary to the terms of data access.
             </li>
           </ol>
-        </div>
-        <div
-          className={css`
-            border-radius: 10px;
-            background-color: #e5f7fd;
-            border: solid 1px ${theme.active};
-            background-color: #${accepted ? `e5f7fd` : `ffffff`};
-            border: solid 1px #${accepted ? `00afed` : `cacbcf`};
-            width: 90%;
-            margin-top: 10px;
-            padding: 10px 25px;
-          `}
-        >
-          <input
-            type="checkbox"
-            checked={accepted}
-            onChange={event => {
-              const { email, percentageFilled, ...rest } = loggedInUser;
-              if (event.target.checked) {
-                setCustomStepMessage(null);
-                trackUserInteraction({
-                  category: TRACKING_EVENTS.categories.join,
-                  action: TRACKING_EVENTS.actions.acceptedTerms,
-                  label: TRACKING_EVENTS.labels.joinProcess,
-                });
-              }
-              setAccepted(event.target.checked);
-              disableNextStep(!event.target.checked);
-              updateProfile(api)({
-                user: {
-                  ...rest,
-                  acceptedTerms: event.target.checked,
-                },
-              }).then(async profile => {
-                await setUser({ ...profile, email, api });
+        </Terms>
+        <CheckboxBubble
+          buffer
+          onClick={active => {
+            const { email, percentageFilled, ...rest } = loggedInUser;
+            if (active) {
+              trackUserInteraction({
+                category: TRACKING_EVENTS.categories.join,
+                action: TRACKING_EVENTS.actions.acceptedTerms,
+                label: TRACKING_EVENTS.labels.joinProcess,
               });
-            }}
-          />
-          <Trans i18nKey="join.terms.userAgreement">
-            I have read and agreed to the Kids First Data Research Portal Term and Conditions
-          </Trans>
-        </div>
-        {customStepMessage && (
-          <div
-            className={css`
-              color: red;
-              width: 90%;
-            `}
-          >
-            {customStepMessage}
```

Comment:
was this just never used? Just wanted to confirm it's safe to remove

Reply:
Yeah good q. Other steps of the wizard disable buttons if you haven't completed what you need to do to go to the next step. For some reason this step was leaving the button enabled and displaying a message that you need to accept terms in order to move forward. Now, just like the other steps we just disable the button until the thing is checked.

It seems more consistent, and the visual feedback is pretty obvious - initially the checkbox bubble is white and the button is disabled / greyed out, when you click the checkbox the checkbox bubble activates in addition to the button turning blue & being enabled

Reply:
uhm yeah that seems better, it kinda feels like a trap otherwise eh :P 
But then I feel like maybe there was a reason why we ended with this behaviour, thinking it might be better to confirm with the BA before we change it, just in case...

Reply:
yeah good call, @rosibaj - you have any thoughts on this?

Reply:
this is a good point @hlminh2000 @jgnieuwhof .  I like your suggestions to keep the button disabled until the terms and conditions are select.